spec/serializers/cache/heading_serializer_spec.rb

```diff
@@ -23,13 +23,16 @@
       goods_nomenclature_sid: heading.goods_nomenclature_sid,
     )
   end
+  let!(:additional_code) { create :additional_code, validity_start_date: 3.years.ago.beginning_of_day }
```

Comment:
AFAIK these are both violations of our linting rules?

I'd just add a trait for these (:with_additional_code)